src/script.ts

```diff
@@ -54,72 +33,143 @@ export const runScript = async (
       'k6'
     )
   } else {
-    // only the architecture directory will be in resources on the packaged app
     k6Path = path.join(process.resourcesPath, getArch(), 'k6')
   }
 
   // add .exe on windows
   k6Path += getPlatform() === 'win' ? '.exe' : ''
 
-  const k6Args = [
-    'run',
-    modifiedScriptPath,
-    '--vus=1',
-    '--iterations=1',
-    '--insecure-skip-tls-verify',
-    '--log-format=json',
-    '--quiet',
-    ...(!enableUsageReport ? ['--no-usage-report'] : []),
-  ]
-
-  // Create symlink to data folder so it can be accessed by the script
-  await symlink(DATA_FILES_PATH, DATA_FILES_TEMP_PATH, 'dir')
-
-  const k6 = spawn(k6Path, k6Args, {
-    env: { ...process.env, ...proxyEnv },
+  const k6 = spawn(k6Path, args, {
+    env: { ...process.env, ...env },
```

Comment:
@Llandy3d do we need the whole `process.env` here?

Reply:
I think we do because we are expanding the default env and want to keep the other values as well 🤔